lib/chef/provider/aws_vpc.rb

```diff
@@ -55,6 +55,17 @@ def update_aws_object(vpc)
   end
 
   def destroy_aws_object(vpc)
+    if purging
```

Comment:
@jkeiser when you attempt to delete a vpc from the UI the user is prompted:
```
Are you sure you want to delete this VPC?. Deleting this VPC will also delete objects associated with this VPC in this region.

Subnets
Security Groups
Network ACLs
VPN Attachments
Internet Gateways
Route Tables
Network Interfaces
VPC Peering Connections
	
Delete VPN Connection when deleting the VPC? (checkbox)
```

We can safely add additional vpc objects to the purge.  If there are running instances networked to the vpc being deleted will the api call not execute? Purge is an excellent work choice if instances must be terminated as well.